task/sql-tasks.js

```diff
@@ -129,7 +197,14 @@ async function task_1_8(db) {
  *
  */
 async function task_1_9(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT 
+	CustomerID,
+    ContactName
+ FROM Customers
+ where ContactName like 'F%___n%'

```

Comment:
n must be exatly 4rth character

Reply:
Yes i know but if i didn't use % after F i didn't get result which i need.